src/drive/mobile/lib/reporter.js

```diff
@@ -2,16 +2,20 @@
 import Raven from 'raven-js'
 
 let isEnable = false
-export const ANALYTICS_URL = `https://${__SENTRY_TOKEN__}@sentry.cozycloud.cc/6`
+export const PROD_ANALYTICS_URL = `https://${__SENTRY_TOKEN__}@sentry.cozycloud.cc/6`
+export const DEV_ANALYTICS_URL = `https://${__SENTRY_TOKEN__}@sentry.cozycloud.cc/2`
```

Comment:
I don't think we need to export these two.

Reply:
:+1: